StarkInfra/StarkInfra/DynamicBrcode/DynamicBrcode.cs

```diff
@@ -0,0 +1,497 @@
+        ///     <item>uuid [string]: unique uuid returned when the DynamicBrcode is created. ex: "4e2eab725ddd495f9c98ffd97440702d"</item>
+        ///     <item>url [string]: url link to the BR Code image. ex: "https://brcode-h.development.starkinfra.com/dynamic-qrcode/901e71f2447c43c886f58366a5432c4b.png"</item>
+        ///     <item>updated [DateTime]: latest update datetime for the DynamicBrcode. ex: DateTime(2020, 3, 10, 10, 30, 0, 0)</item>
+        ///     <item>created [DateTime]: creation datetime for the DynamicBrcode. ex: DateTime(2020, 3, 10, 10, 30, 0, 0)</item>
+        /// </list>
+        /// </summary>
+        public DynamicBrcode( 
+            string name, string city, string externalID, string type=null, 
+            List<string> tags=null, string id=null, string uuid=null, 
+            string url=null, DateTime? updated=null, DateTime? created=null
+        ) : base(id)
+        {
+            Name = name;
+            City = city;
+            ExternalID = externalID;
+            Type = type;
+            Tags = tags;
+            Uuid = uuid;
+            Url = url;
+            Updated = updated;
+            Created = created;
+        }
+
+        /// <summary>
+        /// Create DynamicBrcodes
+        /// <br/>
+        /// Send a list of DynamicBrcode objects for creation at the Stark Infra API
+        /// <br/>
+        /// Parameters (required):
+        /// <list>
+        ///     <item>brcodes [list of DynamicBrcode objects]: list of DynamicBrcode objects to be created in the API.</item>
+        /// </list>
+        /// <br/>
+        /// Parameters (optional):
+        /// <list>
+        ///     <item>user [Organization/Project object, default null]: Organization or Project object. Not necessary if StarkInfra.Settings.User was set before function call</item>
+        /// </list>
+        /// <br/>
+        /// Return:
+        /// <list>
+        ///     <item>list of DynamicBrcode objects with updated attributes</item>
+        /// </list>
+        /// </summary>
+        public static List<DynamicBrcode> Create(List<DynamicBrcode> brcodes, User user = null)
+        {
+            (string resourceName, Api.ResourceMaker resourceMaker) = Resource();
+            return Rest.Post(
+                resourceName: resourceName,
+                resourceMaker: resourceMaker,
+                entities: brcodes,
+                user: user
+            ).ToList().ConvertAll(o => (DynamicBrcode)o);
+        }
+
+        /// <summary>
+        /// Create DynamicBrcodes
+        /// <br/>
+        /// Send a list of DynamicBrcode objects for creation in the Stark Infra API
+        /// <br/>
+        /// Parameters (required):
+        /// <list>
+        ///     <item>brcodes [list of dictionaries]: list of dictionaries representing the DynamicBrcodes to be created in the API</item>
```

Comment:
Para o overload de create tem lugares que esta `list of dictionaries` e outros que estão `list of Dictionaries`. Define um padrão pra deixar todos.